### ISSUE TITLE
Scrollbar gets disabled

### ISSUE THREAD

**Issue by AxharKhan** [themeselection/bootstrap-cheatsheet#5]
I don't know if this is intentional but when I click on an element to see its example, the scrollbar on the page gets disabled and I have to close the current example to view the rest of the cheat sheet.

Tested on: Microsoft Edge Version 107.0.1418.62 (Official build) (64-bit)

**Comment by akaspeh1** [comment]
Yeah, makes the experience worse. Please fix. 10x QoL improvement
